Gemfile.lock

```diff
@@ -186,40 +186,41 @@ GEM
       dotenv (= 3.1.8)
       railties (>= 6.1)
     drb (2.2.1)
-    dry-configurable (0.16.1)
-      dry-core (~> 0.6)
+    dry-configurable (1.3.0)
```

Comment:
Where is this actually used in the code base?

Reply:
Settings loading.

Reply:
This type of stuff https://github.com/CommitChange/houdini/blob/supporter_level_goal/app/legacy_lib/onboard_accounts.rb


I assume our goal is to eventually replace all the app/legacy_lib code with regular ActiveModel/Record/etc... Correct? (Just wanting to make sure I understand the long-term vision).